### ISSUE TITLE
[Feature Request] Rendered images with links

### ISSUE THREAD

**Issue by rtakehara** [CalebJohn/joplin-rich-markdown#45]
It would be really useful to be able to put images on links, for example:

```markdown

[![](image_url)](link_url)

```

to make something like this:

[![](https://github.com/CalebJohn/joplin-rich-markdown/blob/main/examples/welcome.png?raw=true)](https://github.com/CalebJohn/joplin-rich-markdown)

Is it doable or does it end up working like an inline image since it is in the same line as a link?

**Comment by CalebJohn** [comment]
I think this will be doable, it might be a fairly simple fix. I'll take a look at it tonight.

**Comment by CalebJohn** [comment]
> Is it doable or does it end up working like an inline image since it is in the same line as a link?

In versions 0.10.0 and lower it worked like an inline image. I've now added some extra code to handle this as a special case, so now in version 0.11.0 works the way you've requested. I also found and fixed an related bug while implementing this!

v0.11.0 has just been released and should be available for you to install shortly. Thanks for the request!
